### PR TITLE
Blog Post: LTS For Mike Dilger

### PR DESCRIPTION
**Pull request by Arvin21M**
**Build Preview:**
- [/blog/mike-dilger-receives-lts-grant](https://os-website-git-michael-dilger-nostr-lts-open-sats.vercel.app/blog/mike-dilger-receives-lts-grant)

**Social Preview:**
- image

### CODE REVIEW

**Comment by mikedilger** [comment]
I wouldn't include "by leveraging Tor" even though I recommend people run gossip behind Tor.  Just because gossip doesn' thave any Tor-specific code, doesn't use tor proxy, etc, and I wouldn't want people to presume it has such code.  I might change the last part of that sentence to "a nostr client designed to prioritize privacy and security by avoiding [common web stack vulnerabilities](https://owasp.org/www-project-top-ten/#) and leveraging rust language security."

As for my continued plans, things have progressed since my grant application and I'm not sure the list should be as it was or be updated.  Your call.  In any case:
* Viewing reactions on posts is in, as well as reacting via emoji
* NIP-46 as a client unfortunately hit a dead end due to a compiler bug I have been unable to work around, so that is shelved.
* Bookmarks are in
* Relay global is in

I like my quote as is.  I do remember saying that.

**Comment by Arvin21M** [comment]
Thank you for the updates on the project progress @mikedilger, and for clearing up the Tor statement. I implemented these changes into the draft. That's a great quote!

If you think of any other changes, let me know. 🙏

**Comment by Arvin21M** [comment]
@OpenSats/volunteers @Bayernatoor @btcschellingpt @moteanup24
Would some one please give this a quick sanity check for any changes we didn't catch or know we needed?

**Review by btcschellingpt — APPROVED** [review]
@Arvin21M looks good and reads well and all links working .. it's ready to go, so make it so!

File: data/blog/mike-dilger-receives-lts-grant.mdx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+contributor in the nostr ecosystem.
+
+Mike is known for being the lead developer of
+[Gossip](https://github.com/mikedilger/gossip?tab=readme-ov-file#), a nostr
+client designed to prioritize privacy and security by avoiding [common web stack
+vulnerabilities](https://owasp.org/www-project-top-ten/#) and leveraging rust
+language security.
+
+Gossip has become a reference implementation for the "[outbox
+model](https://mikedilger.com/gossip-model/)", a design that enables more
```
Comment (dergigi) [inline data/blog/mike-dilger-receives-lts-grant.mdx:21]:
```suggestion
model](https://mikedilger.com/gossip-model/)," a design that enables more
```

File: data/blog/mike-dilger-receives-lts-grant.mdx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+scalable message distribution across the nostr network. In addition to Gossip,
+Mike has contributed to several tools and libraries that support the nostr
```
Comment (dergigi) [inline data/blog/mike-dilger-receives-lts-grant.mdx:23]:
```suggestion
Mike is known for his contributions to several tools and libraries that support the nostr
```

File: data/blog/mike-dilger-receives-lts-grant.mdx [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+ecosystem, such as the [chorus relay](https://github.com/mikedilger/chorus),
+[relay tester](https://github.com/mikedilger/relay-tester), and his active
```
Comment (dergigi) [inline data/blog/mike-dilger-receives-lts-grant.mdx:25]:
```suggestion
[relay tester](https://github.com/mikedilger/relay-tester), as well as his active
```

File: data/blog/mike-dilger-receives-lts-grant.mdx [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+[participation in NIP
+discussions](https://github.com/nostr-protocol/nips/issues?q=involves%3Amikedilger).
+
+With the support of this LTS grant, Mike plans to continue his work on the
+Gossip client, with a focus on implementing key features such as:
```
Comment (dergigi) [inline data/blog/mike-dilger-receives-lts-grant.mdx:30]:
```suggestion
Gossip client, focusing on:
```

File: data/blog/mike-dilger-receives-lts-grant.mdx [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+- Improving support for replying to various versions of replaceable events.
+
+- Adding [NIP-96](https://github.com/nostr-protocol/nips/blob/master/96.md) file
+  uploads.
+
+- Implementing Blossom support, Web of Trust, and search at relay features.
```
Comment (dergigi) [inline data/blog/mike-dilger-receives-lts-grant.mdx:37]: > search at relay features.

I'm not sure what that means?

Comment (Arvin21M) [reply data/blog/mike-dilger-receives-lts-grant.mdx:37]: I was not sure initially when writing the first draft. It was part of the original source material. But in reading it again before opening the PR, I was fairly certain that it means having search capabilities at the relay level, to be able to search events from a specific relay. Perhaps I should reword this to "...and relay search functionality."

File: data/blog/mike-dilger-receives-lts-grant.mdx [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+
+With the support of this LTS grant, Mike plans to continue his work on the
+Gossip client, focusing on:
+
+- Improving support for replying to various versions of replaceable events.
+
+- Adding [NIP-96](https://github.com/nostr-protocol/nips/blob/master/96.md) file
+  uploads.
+
+- Implementing Blossom support, Web of Trust, and search functionality at relay level.
```
Comment (dergigi) [inline data/blog/mike-dilger-receives-lts-grant.mdx:37]:
```suggestion
- Implementing Blossom support, Web of Trust, and search functionality at the relay level.
```

File: data/blog/mike-dilger-receives-lts-grant.mdx [hunk 7]
```diff
@@ -0,0 +1,57 @@
+---
+title: "Long-Term Support For Mike Dilger"
+date: '2024-09-15'
```
Comment (dergigi) [inline data/blog/mike-dilger-receives-lts-grant.mdx:3]:
```suggestion
date: '2024-09-11'
```

File: data/blog/mike-dilger-receives-lts-grant.mdx [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+[Gossip](https://github.com/mikedilger/gossip?tab=readme-ov-file#), a nostr
+client designed to prioritize privacy and security by avoiding [common web stack
+vulnerabilities](https://owasp.org/www-project-top-ten/#) and leveraging rust
+language security.
+
+Gossip has become a reference implementation for the "[outbox
+model](https://mikedilger.com/gossip-model/)," a design that enables more
+scalable message distribution across the nostr network. In addition to Gossip,
+Mike is known for his contributions to several tools and libraries that support the nostr
+ecosystem, such as the [chorus relay](https://github.com/mikedilger/chorus),
```
Comment (dergigi) [inline data/blog/mike-dilger-receives-lts-grant.mdx:24]:
```suggestion
ecosystem, such as the [chorus relay](https://github.com/mikedilger/chorus) and
```

**Review by dergigi — APPROVED** [review]
Double-checked everything and made some minor adjustments, including publication date. 

Looks good to me! ✅

:shipit:

**Pull request merged**